tests/sles4sap/sapconf.pm

```diff
@@ -113,6 +106,12 @@ sub verify_sapconf_service {
 
 sub run {
     my ($self) = @_;
+    my @tuned_profiles = is_sle('>=15') ?
```

Comment:
```suggestion
    my @tuned_profiles = qw(balanced desktop latency-performance network-latency network-throughput
      powersave sapconf saptune throughput-performance virtual-guest virtual-host);
     push @tuned_profile, qw(sap-ase sap-bobj sap-hana sap-netweaver) unless is_sle('>=15');
```

look like second list is actually extension of first , I think this way looks more clear what you actually add in each case

Reply:
Actually, the order of the profiles within the list is important, as the list is turned into a regexp in L131 and used to check the output of `tuned-adm list`

Reply:
ah now I remember that we already had this discussion in some of your previous PRs :) 
sorry will try keep this in mind next time :)